.github/workflows/developer.yml

```diff
@@ -52,6 +52,9 @@ jobs:
     - name: test
       run: |
         cd bufr/build
+        cd test
+        bash -x run_test_bort.sh
+        cd ..
         ctest --verbose --output-on-failure --rerun-failed
```

Comment:
Why is run_test_bort.sh now being run directly in the developer.yml script?  There's already an add_test directive in CMakeLists.txt which runs the run_test_bort.sh script for all of the builds, so we now end up running it twice in the developer CI build.  Why is this necessary?

Reply:
I will fix. this was an artifact of debugging...